docs/articles/new-expensify/billing-and-subscriptions/Billing-page.md

```diff
@@ -1,6 +1,49 @@
 ---
 title: Billing and Subscriptions
-description: An overview of how billing works in Expensify.
+description: Billing Page Overview
 ---
 
-# Coming Soon
+# Billing Overview
+
+At the beginning of each month, the Billing Owner for the workspace will be billed for the previous month’s activity. 
+Your Expensify bill is determined by: 
+- The number of active members in your workspace
+- Whether you have a Collect or Control plan
+- Whether you’re on pay-per-use or an annual subscription
+- Whether you’re using the Expensify Visa® Commercial Card
+- Active members
```

Comment:
This is a bit redundant because the first bullet says "The number of active members in your workspace", so I think we can remove it, but it's not a blocker
```suggestion
```